lib/gutenberg/front-end.css

```diff
@@ -44,61 +44,61 @@ h6.light {
 /* Color Palette
 ---------------------------------------------------------------------------- */
 
-.entry-content .has-light-gray-background-color {
+.site-container .has-light-gray-background-color {
 	background-color: #f5f5f5 !important;
 }
 
-.entry-content .has-light-gray-color {
+.site-container .has-light-gray-color {
 	color: #f5f5f5 !important;
 }
 
-.entry-content .has-medium-gray-background-color {
+.site-container .has-medium-gray-background-color {
 	background-color: #999 !important;
 }
 
-.entry-content .has-medium-gray-color {
+.site-container .has-medium-gray-color {
 	color: #999 !important;
 }
 
-.entry-content .has-dark-gray-background-color {
+.site-container .has-dark-gray-background-color {
 	background-color: #333 !important;
 }
 
-.entry-content .has-dark-gray-color {
+.site-container .has-dark-gray-color {
 	color: #333 !important;
 }
 
 /* Background Color
 ---------------------------------------------------------------------------- */
 
-.entry-content p.has-background {
+.site-container p.has-background {
 	padding: 25px 30px;
 }
 
-.entry-content p.has-background.box-shadow {
+.site-container p.has-background.box-shadow {
 	box-shadow: 0 0 20px rgba(0, 0, 0, 0.1);
 }
 
-.entry-content p.has-text-color a,
-.entry-content p.has-background a {
+.site-container p.has-text-color a,
+.site-container p.has-background a {
 	color: inherit;
 }
 
-.entry-content p.has-text-color a:focus,
-.entry-content p.has-background a:focus,
-.entry-content p.has-text-color a:hover,
-.entry-content p.has-background a:hover {
+.site-container p.has-background a:focus,
+.site-container p.has-background a:hover,
+.site-container p.has-text-color a:hover,
```

Comment:
@dreamwhisper Should `.has-text-color a:focus` appear before the hover here?

Reply:
@nickcernis corrected. I should just copy/paste from others I've fixed at this point :)